spec/chewy/fields/base_spec.rb

```diff
@@ -405,15 +405,8 @@
         stub_model(:city)
         stub_model(:country)
 
-        case adapter
-        when :active_record
-          City.belongs_to :country
-          if ActiveRecord::VERSION::MAJOR >= 4
-            Country.has_many :cities, -> { order :id }
-          else
-            Country.has_many :cities, order: :id
-          end
-        end
+        City.belongs_to :country
+        Country.has_many :cities, order: :id
```

Comment:
We're expecting ActiveRecord >= 4, aren't we?
```suggestion
        Country.has_many :cities, -> { order :id }
```

Reply:
Thank you so much! @rabotyaga